test/tests/testSendNotification.js

```diff
@@ -170,6 +170,8 @@ suite('sendNotification', function() {
     .then(function() {
       assert(false, 'sendNotification promise resolved');
     }, function() {
```

Comment:
derp - sorry, didn't know how to run these tests locally, figured I'd let travis help me out ;)